profile/profile.md

```diff
@@ -0,0 +1,71 @@
+# Roshan Ramesh . Jarvis Consulting
```

Comment:
this file should be README.md..

make sure you set up the latest script 
https://github.com/jarviscanada/jarvis_profile_builder/blob/master/profile_app.sh